src/sentry/models/eventattachment.py

```diff
@@ -70,7 +87,16 @@ def delete(self, *args, **kwargs):
             # repopulated with the next incoming crash report.
             cache.delete(get_crashreport_key(self.group_id))
 
+        if self.blob_path:
+            if self.blob_path.startswith("eventattachments/v1/"):
+                storage = get_storage()
+
+            storage.delete(self.blob_path)
```

Comment:
Is `storage` defined when the `if` branch above does not execute?

Reply:
I added an explicit `NotImplementedError` in an else branch to avoid this.